pkg/types/vsphere/defaults/platform.go

```diff
@@ -14,10 +15,43 @@ func SetPlatformDefaults(p *vsphere.Platform, installConfig *types.InstallConfig
 	// but _not_ control plane nodes. If the placementConstraints
 	// are not defined we must use the default for the datacenter
 	// and cluster.
```

Comment:
this comment needs to be revised, placementConstraints were refactored out

Reply:
Oh yes, thanks. I'll get that fixed up.

Reply:
I removed the comment since its not obvious to me how the OVA upload matters that early in install-config validation. Hope that's alright.